lightning/src/ln/channelmanager.rs

```diff
@@ -1057,9 +1057,9 @@ impl<ChanSigner: ChannelKeys, M: Deref, T: Deref, K: Deref, F: Deref> ChannelMan
 
 		channel_state = Some(self.channel_state.lock().unwrap());
 		if let &PendingHTLCStatus::Forward(PendingHTLCInfo { ref forward_or_receive, ref amt_to_forward, ref outgoing_cltv_value, .. }) = &pending_forward_info {
-			// If short_channel_id is 0 here, we'll reject them in the body here (which is
-			// important as various things later assume we are a ::Receive if short_channel_id is
-			// non-0.
+			// If short_channel_id is 0 here, we'll reject them as there cannot be a channel with a
```

Comment:
Is 'them' referring to the HTLC (or the onion more broadly)? Consider spelling it out as such if so.